interbotix_xs_toolbox/interbotix_xs_modules/interbotix_xs_modules/xs_robot/turret.py

```diff
@@ -182,14 +183,16 @@ def __init__(
         )
 
         while rclpy.ok() and not self.future_group_info.done():
-            rclpy.spin_until_future_complete(self.core, self.future_group_info)
-            rclpy.spin_once(self.core)
+            rclpy.spin_until_future_complete(self.core.get_node(), self.future_group_info)
+            rclpy.spin_once(self.core.get_node())
 
         group_info: RobotInfo.Response = self.future_group_info.result()
         self.pan_name = group_info.joint_names[0]
         self.tilt_name = group_info.joint_names[1]
-        pan_limits = [group_info.joint_lower_limits[0], group_info.joint_upper_limits[0]]
-        tilt_limits = [group_info.joint_lower_limits[1], group_info.joint_upper_limits[1]]
+        pan_limits = [group_info.joint_lower_limits[0],
```

Comment:
Please revert all changes like this one that do not affect how the program works. Indentation fixes are okay to keep in.